package.json

```diff
@@ -67,6 +67,7 @@
     "aguid": "^2.0.0",
     "eslint": "^7.0.0-alpha.0",
     "eslint-plugin-prettier": "^3.1.2",
+    "ms": "^2.1.1",
```

Comment:
`ms` hasn't been required by the `pre-commit` hook in the past ... 💭 
What changed?

Reply:
```suggestion
```
Can we attempt to see if Travis-CI will build the project without `ms`?

Reply:
That's very bizarre. 😕 
Surely if it's a ("child") dependency of `eslint` it should be installed by `eslint` ... 💭 
Thanks very much for checking and removing it. 👍